api/v3/Contract/ProcessScheduledModifications.php

```diff
@@ -81,6 +82,18 @@ function civicrm_api3_Contract_process_scheduled_modifications($params) {
     // execute the changes
     $change = CRM_Contract_Change::getChangeForData($scheduled_activity);
     $result['order'][] = $change->getID();
+
+    $requestedExecutionTime = strtotime($scheduled_activity["activity_date_time"]);
```

Comment:
I think using the current time might make more sense here. Thinking of a scenario like:
- Cancellation is entered with `2020-01-01`
- There was another scheduled change, so the activity was set to "Needs Review"
- `2020-01-05` is a debit date, so someone sets `contract_minimum_change_date` to `2020-01-06`
- On that day, someone works through the "Needs Review" activities, deletes the other scheduled change and sets the cancellation back to "Scheduled". Date remains as `2020-01-01`.
- Shortly after, `Contract.process_scheduled_modifications` picks up that change and executes it (which we wouldn't want to happen)